test/tags.js

```diff
@@ -21,13 +21,13 @@ describe('Custom Tags', function() {
     expect(engine.render('/views/content')).to.equal('/dist/images/calvin.png');
   });
 
-  it('allows custom tags that do not process the first arg as a path', function() {
+  it('allows tags that do not process the first arg as a path', function() {
     const engine = beard({
       templates: {
-        '/view': `{{tag 'a', {href: 'www.google.com', link: 'google'}}}`
+        '/view': `{{beardTag 'a', {href: 'www.google.com', link: 'google'}}}`
       },
-      customTags: {
-        tag: {
+      tags: {
+        beardTag: {
```

Comment:
@joeosburn why is this named `beardTag` here instead of `tag`?

Reply:
It is an old "custom tag" implementation for a test from back when we didn't have a default `tag` tag. It can't conflict with our actual `tag` implementation, so I just renamed it to something simply. If this is too confusing, I can just update the test entirely to not use a tag-like test "custom" tag.